extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -561,6 +566,30 @@ export class AmpConsent extends AMP.BaseElement {
     }
   }
 
+  /**
+   * Check if `exposes` API has been set from CMP and merge with inline.
+   * Expose the proper APIs, if applicable.
+   */
+  exposeApis_() {
+    this.getConsentRemote_().then((response) => {
+      const remoteExposes = response ? validateExposesApi(response) : [];
```

Comment:
what happens if a message is received beforehand?

Reply:
Ahh yes, that's a good point. After our convo today, I think we don't have to allow CMP to send this signal via `checkConsentHref`, instead they can just have it in their checked in config or request that pubs add it themselves. WDYT?

Reply:
sounds reasonable to me. Whether to support the API or not should be a static decision, and there's no need to get it remotely.